.github/workflows/pr_push.yml

```diff
@@ -48,6 +48,31 @@ jobs:
         - name: Build
           run: >
             cmake --build ${{github.workspace}}/build --config Release -j ${{matrix.nproc}}
+
+    CodeStyle:
+      name: Coding style
+      runs-on: ubuntu-latest
+
+      steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install apt packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y cmake clang-format-15
+
+      - name: Configure CMake
+        run: >
+          cmake
+          -B ${{github.workspace}}/build
+          -DUMF_FORMAT_CODE_STYLE=ON
+
+      - name: Build
+        run: >
+          cmake --build build --target clang-format-check  
+
```

Comment:
I can see that clang-format-check is run on Mac (in basic.yml) - should we get rid of it now...?

Reply:
Done